modules/editable-layers/package.json

```diff
@@ -70,6 +70,7 @@
     "viewport-mercator-project": ">=6.2.3"
   },
   "peerDependencies": {
+    "@deck.gl-community/layers": "^9.0.2",
```

Comment:
In deck.gl we have the peerDependencies defined more loosely, e.g. `"^9.0.0"`, any reason not to do that in this repo also?

Reply:
@felixpalmer Thanks for the help. I relaxed the version constraint according to your suggestion.-